assets/pages/bundles-landing/helpers/subscriptionsLinks.js

```diff
@@ -25,14 +25,16 @@ const defaultPromos: PromoCodes = {
 
 const tests = [
   {
-    name: 'rebaseline',
+    name: 'baseline',
     intCmps: [
-      'gdnwb_copts_memco_sandc_epic_rebaseline_support_proposition_support_proposition',
+      'gdnwb_copts_memco_sandc_epic_baseline_support_proposition',
+      'gdnwb_copts_memco_sandc_header_baseline_support_proposition',
+      'gdnwb_copts_memco_sandc_engagement_banner_baseline_support_proposition',
```

Comment:
These will need to include the variant part of the INTCMP e.g. `gdnwb_copts_memco_sandc_epic_baseline_support_proposition_control` or `gdnwb_copts_memco_sandc_epic_baseline_support_proposition_variant_one`.